dj/models/node.py

```diff
@@ -152,6 +152,44 @@ class NodeMissingParents(SQLModel, table=True):  # type: ignore
     )
 
 
+class AvailabilityStateBase(SQLModel):
+    """
+    An availability state base
```

Comment:
Does this also need a `last_updated_at` timestamp? To indicate when this availability state was last posted.

Reply:
I can add that, is there a specific feature that you think could use that or is it more for having operational information to present to the user? If it's the later I'm wondering if the `valid_through_ts` is a better indicator for users. I'm worried that even a backfill of a single day far in the past may technically update a `last_updated_at` value and create the appearance the data is processed up to that timestamp.

Reply:
Agree with @shangyian that `updated_at` would be useful to know, if nothing else then for "operational reasons" ;) But we don't need the `last_` prefix though.

Reply:
Updated the PR to add an `updated_at` attribute. I also added a test as a sanity check to make sure that through multiple calls it's actually updated. 😄